### ISSUE TITLE
old instances still can have entities without templates.

### ISSUE THREAD

**Issue by daneryl** [huridocs/uwazi#3519]
migration to add the default template ?

**Comment by txau** [comment]
@daneryl I thought we already did a migration for this (migration 9, default templates). Does it mean that this migration failed at setting some of them?

**Comment by daneryl** [comment]
@txau did not remembered this, maybe it failed, or maybe there is another condition where this still can be happening.

**Comment by RafaPolit** [comment]
That migration only ensures that there is template configured as default.  It does not assign that default template to entities that could have been created just by uploading documents prior to the default template procedure.

**Comment by txau** [comment]
> That migration only ensures that there is a template configured as `default`. It does not assign that default template to entities that could have been created just by uploading documents prior to the default template procedure.

I guess the reason for that is to avoid confusing users that had some entities without template "waiting" to be processed. Now if there are user who still have these template-less entities I guess are abandoned entities, old instances, etc.

**Comment by RafaPolit** [comment]
Yes, old instances with uploads of PDFs that never got assigned a template.  They shouldn't exist at this point.

**Comment by daneryl** [comment]
there is a lot of instances we are hosting in this state, one of them with more than 2k entities without template.

**Comment by txau** [comment]
@daneryl I guess instead of the default template, we could inject a new template "Missing template", without metadata except the title so we don't mess up with current instance data.

**Comment by fnocetti** [comment]
@txau @daneryl @RafaPolit is this still relevant?

**Comment by txau** [comment]
I haven't seen errors related to this in a while. Let's assume it is fixed and open a new issue if we actually encounter the problem again.
